src/a13-dates/test/solution.test.js

```diff
@@ -0,0 +1,20 @@
+import assert from 'node:assert'
+import { getCurrentFormattedDate } from '../src/index.js'
+import { test, mock } from 'node:test'
+
+test('getCurrentFormattedDate returns the correct format', () => {
+  // Mock Date to a specific timestamp
```

Comment:
do we need all these comments? I think the code is pretty clear already

Reply:
We can get rid of them, but I don't see it as a big problem since the main purpose of this code is mainly learning